### PR TITLE
DM-3560: Updated crisis line modal styling and info

### PR DESCRIPTION
**Pull request by JoshingYou1**
### JIRA issue link
https://agile6.atlassian.net/browse/DM-3560

## Description - what does this code do?
Makes a few style and content changes to the crisis line modal. Refer to the JIRA link for the content changes.

## Testing done - how did you test it/steps on how can another person can test it 
1. Visit the homepage. Click on the crisis line modal button at the top right of the page. Make sure the styles match the Figma designs. **Changes based on design's feedback:** External links can have the external link styling (the arrow icon at the end of the link). The period at the end of the `Get more resources at...` line should be removed, due to having the external link icon.

## Link to mock-ups/mock ups (image file if you have it) (if applicable)
https://www.figma.com/file/dkvSpADVqwN5KKByF1TVIa/Design-System---VA-DM?node-id=1305%3A3&t=Qb6SB63cNW5OlNfH-0

## Acceptance criteria
- Devs build phone # and chat changes
- Have design review
- Release to all three environments

## Definition of done
- [ ] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [X] A link has been provided to the originating JIRA issue
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by bradjohnson92008 — APPROVED** [review]
LGTM Thanks Josh!

File: app/views/shared/_veterans_crisis_line_modal.html.erb [hunk 1]
```diff
@@ -3,43 +3,60 @@
...
+            </h3>
           </div>
           <p class="line-height-26 margin-bottom-3" id="va-crisis-line-info">
             If you are a Veteran in crisis or concerned about one, connect with our caring, qualified responders for confidential help. Many of them are Veterans themselves.
           </p>
           <ul class="va-crisis-line-contact-list padding-0 margin-0 line-height-26">
             <li class="display-flex flex-align-center margin-bottom-3">
               <i class="fas fa-phone fa-2x margin-right-2 text-center"></i>
-              <a href="tel:18002738255" aria-label="Call the Veterans Crisis Line" class="usa-link">Call 800-273-8255 and select 1</a>
+              <a href="tel:988" aria-label="Call the Veterans Crisis Line" class="usa-link">
```
Comment (camillevilla) [inline app/views/shared/_veterans_crisis_line_modal.html.erb:24]:
```suggestion
              <a href="tel:988" aria-label="Call the Veterans Crisis Line, Call 988 and select 1" class="usa-link">
```

File: app/views/shared/_veterans_crisis_line_modal.html.erb [hunk 2]
```diff
@@ -3,43 +3,60 @@
...
+                Call <b>988 and select 1</b>
+              </a>
             </li>
             <li class="display-flex flex-align-center margin-bottom-3">
-              <i class="fas fa-mobile-alt fa-2x margin-right-2 text-center"></i><a href="sms:838255" aria-label="Text the Veterans Crisis Line" class="usa-link">Text 838255</a>
+              <i class="fas fa-mobile-alt fa-2x margin-right-2 text-center"></i>
+              <a href="sms:838255" aria-label="Text the Veterans Crisis Line" class="usa-link">
```
Comment (camillevilla) [inline app/views/shared/_veterans_crisis_line_modal.html.erb:30]:
```suggestion
              <a href="sms:838255" aria-label="Text the Veterans Crisis, Text 838255" class="usa-link">
```

**Review by camillevilla — CHANGES_REQUESTED** [review]
This is looking good, @JoshingYou1! Can you make two small tweaks to the `aria-label` for the phone line and text number? Right now the numbers in the `<a>` text aren't getting read out to screenreaders because  the `aria-label` is overriding it. Adding the numbers will make it easier if a user wants to record the numbers and contact the crisis line on a different device.

File: app/views/shared/_veterans_crisis_line_modal.html.erb (hunk 1 above)
Comment (JoshingYou1) [reply app/views/shared/_veterans_crisis_line_modal.html.erb]: Done!

File: app/views/shared/_veterans_crisis_line_modal.html.erb (hunk 2 above)
Comment (JoshingYou1) [reply app/views/shared/_veterans_crisis_line_modal.html.erb]: Done!

**Review by camillevilla — APPROVED** [review]
🎶Music to my ears 😎

**Pull request merged**